### DISCUSSION TITLE
Career Portal says: An error has occurred while loading. Please refresh the page and try again.

### DISCUSSION THREAD

**Discussion by CIBR1** [bullhorn/career-portal#452] [Q&A] [answered]
INSERT SHORT DESCRIPTION EXPLAINING THE HIGH-LEVEL REASON FOR THE NEW ISSUE HERE.

## Current behavior

-We're rebuilding our site with a different builder. We copied all the career portal settings, installed the same plugin, used the same shortcode but our career portal shows this message: An error has occurred while loading. Please refresh the page and try again.

## Expected behavior

- We need the career portal to show the data we have on the live site.

## Steps to replicate behavior

1.

## Screenshots

__Current__
https://cibrrebuild.wpengine.com/careers/

__Expected__
https://cibrwarriors.com/careers

**Comment by charlesabarnes** [comment, marked as answer]
In the new site config, there is a field "referredBy" that was not in the previous website config.  Removing that will make it work.

**Reply by CIBR1** [reply]
this totally worked! Thank you very much!
